dataplane/worker.h

```diff
@@ -74,7 +74,7 @@ class cWorker
 
 	YANET_NEVER_INLINE void mainThread();
 
-	inline void calcHash(rte_mbuf* mbuf);
+	inline void calcHash(rte_mbuf* mbuf, bool without_ports = false);
```

Comment:
Is this acceptable?

Reply:
I definitely do not like the parameter naming. Also I would suggest to use a bitmask here like `L3_HASH | L4_HASH` and `L3_HASH` or something other like this.